scripts/prepend-go-build-to-tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# prepend-go-build-to-tests.sh prepends '//go:build <BUILD_TAG>\n\n' to '*_test.go' files within
+# the directory it is run as well as subdirectories as long as those files don't already starts
+# with '//go:build' or '// Code generated by'
+
+usage() {
+  echo "usage: $0 <BUILD_TAG>"
+  exit 2
+}
+
+BUILD_TAG=$1
+if [[ -z $BUILD_TAG ]]; then
+  usage
+fi
+
+for file in $(git ls-files -- '*_test.go'); do
+	if [[ $(sed -n '/^\/\/go:build/p;q' "${file}") ]]; then
+		#echo "${file} starts with //go:build"
```

Comment:
What should be done about the leftover debug output? I would say: Either convert them to log lines or remove them.

Reply:
Just checked through our shell scripts and the closest we do to logging is

```bash
info() {
    echo "INFO: $(date): $*"
}
```
But leaving echos in when running the script over subdirectories with large amounts of files is too verbose, I'll just remove both commented out echos then.